go-controller/pkg/ovn/egressip.go

```diff
@@ -2025,6 +1987,54 @@ func createDefaultNoRerouteServicePolicies(nbClient libovsdbclient.Client, v4Clu
 	return nil
 }
 
+// createDefaultRouteToExternal is called only when IC=true. This function adds a "catch-all" kind of LRSR to ovn-cluster-router
```

Comment:
ive a question for @trozet / surya for pod2pod comms where theres a local pod and remote pod.
With this new static route, when theres no static route match besides this new static route, it will send traffic out LR port `rtoj-ovn_cluster_router`. 

 I am curious how a remote pod trying to comm with a local pod works with this. 
Isnt this new static route overriding how the LR "learns" networks attached to it on local switches (i.e. the local pod network attached to port `rtos-ovn-${switchname}` ). 
Can someone walk me through the steps how the LR chooses the port to forward it out to in this scenario.
I am not doubting this works. Ive tested it.
It looks to me it gets sent out LR port `rtoj-ovn_cluster_router` but it should be sent out port `rtos-ovn-${switchname}`.

Reply:
Surya explained to me over slack. Learning from local attached networks/switches has greater precedence than these static routes. I thought the converse was true.
She will attach ovn trace here eventually to show the decision process by the LR.

Reply:
```
 1. lr_in_lookup_neighbor (northd.c:11956): 1, priority 0, uuid 7f697148
    reg9[2] = 1;
    next;
 2. lr_in_learn_neighbor (northd.c:11965): reg9[2] == 1 || reg9[3] == 0, priority 100, uuid 9a51447b
    next;
12. lr_in_ip_routing_pre (northd.c:12190): 1, priority 0, uuid 7098d87a
    reg7 = 0;
    next;
13. lr_in_ip_routing (northd.c:10603): ip4.dst == 10.244.1.0/24, priority 74, uuid d6770c72
    ip.ttl--;
    reg8[0..15] = 0;
    reg0 = ip4.dst;
    reg1 = 10.244.1.1;
    eth.src = 0a:58:0a:f4:01:01;
    outport = "rtos-ovn-worker2";
    flags.loopback = 1;
    next;
```
adding this here for more context,